javaagent-tooling/javaagent-tooling-java9/src/main/java/io/opentelemetry/javaagent/tooling/ExposeAgentBootstrapListener.java

```diff
@@ -52,7 +54,12 @@ private void exposeModule(JavaModule fromModule, JavaModule targetModule) {
     if (fromModule != JavaModule.UNSUPPORTED
         && fromModule.isNamed()
         && !fromModule.canRead(targetModule)) {
-      logger.debug("Adding module read from {} to {}", fromModule, targetModule);
+
+      logger.log(
+          Level.FINE,
+          "Adding module read from {0} to {1}",
+          new Object[] {fromModule, targetModule});
```

Comment:
For this pattern we should probably check loggable to avoid array allocation when disabled

Reply:
Good point, done.